integrations/LinkedInInsightTag/browser.js

```diff
@@ -18,6 +20,9 @@ class LinkedInInsightTag {
     if (!this.partnerId) {
       return;
     }
+    if (this.eventConversionId === undefined) {
+      this.eventConversionId = [];
+    }
```

Comment:
No need to check it here. We are already checking this inside `getHashFromArrayWithDuplicate` function.

Reply:
updated